carmm/run/aims_path.py

```diff
@@ -57,40 +57,56 @@ def set_aims_command(hpc='hawk', basis_set='light', defaults=2010, nodes_per_ins
     """Set the relevant environment variables based on HPC"""
     os.environ["AIMS_SPECIES_DIR"] = fhi_aims_directory[hpc] + species
 
-    no_cpus = {"hawk": 40,
-              "hawk-amd": 64,
-              "archer2": 128,
-              "isambard": 64,
-              "young":64,
-              "aws": 72}
-
+    cpus_per_node = {
+        "hawk": 40,
+        "hawk-amd": 64,
+        "archer2": 128,
+        "isambard": 64,
+        "young":64,
+        "aws": 72
+    }
 
+    if hpc in ["hawk","hawk-amd"]:
+        requested_tasks = os.environ["SLURM_NTASKS"]
+        requested_nodes = os.environ["SLURM_NNODES"]
```

Comment:
Those variables must be set up in the test, otherwise they will throw an error outside of an HPC environment

Reply:
Added a try/except, rather than setting the variable for testing, as this makes it more bulletproof. It's only a check anyway, I might separate into a different function.